src/utils/media/effects/virtual-background/VideoStreamBackgroundEffect.js

```diff
@@ -0,0 +1,667 @@
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
```

Comment:
We're keeping an original date of submitting, if files were renamed and then modified

```suggestion
 * SPDX-FileCopyrightText: 2021 Nextcloud GmbH and Nextcloud contributors
```